ni/measurementlink/sessionmanagement/v1/session_management_service.proto

```diff
@@ -73,6 +73,9 @@ service SessionManagementService {
 
   // Gets all multiplexer sessions currently registered with this service.
   rpc GetAllRegisteredMultiplexerSessions(GetAllRegisteredMultiplexerSessionsRequest) returns (GetAllRegisteredMultiplexerSessionsResponse);
+
+  // Gets all the reserved, registered instrument sessions and registered multiplexer sessions.
```

Comment:
Consider updating to:

Gets all reserved or registered sessions for the specified instrument and multiplexer types.